### ISSUE TITLE
[Bug]: DeprecationWarning triggered from using distutils

### ISSUE THREAD

**Issue by johnthagen** [matplotlib/matplotlib#21643]
### Bug summary

distutils is deprecated in Python 3.10: https://docs.python.org/3/library/distutils.html

Matplotlib (as of 3.4.3) currently uses `distutils`, so users are met with `DeprecationWarning`s when using on Python 3.10.

```
    from matplotlib import pyplot 
.tox/py/lib/python3.10/site-packages/matplotlib/__init__.py:88: in <module>
    from distutils.version import LooseVersion
/usr/local/lib/python3.10/distutils/__init__.py:19: in <module>
    warnings.warn(_DEPRECATION_MESSAGE,
E   DeprecationWarning: The distutils package is deprecated and slated for removal in Python 3.12. Use setuptools or check PEP 632 for potential alternatives
```

### Code for reproduction

```python
from matplotlib import pyplot
```

### Actual outcome

`DeprecationWarning` raised per summary.

### Expected outcome

No `DeprecationWarning`s raised

### Operating system

macOS

### Matplotlib Version

3.4.3

### Python version

Python 3.10.0

### Installation

pip

**Comment by dstansby** [comment]
Thanks for reporting. I think this is a duplicate of https://github.com/matplotlib/matplotlib/issues/21057, which has been fixed.

**Comment by johnthagen** [comment]
Yes, it does appear that this particular import is gone in `main`:

https://github.com/matplotlib/matplotlib/blob/993c1bc1b9e338ce6479c559105322a838a78d33/lib/matplotlib/__init__.py#L84-L90

**Comment by johnthagen** [comment]
Fixed in #20437
